--- tecsgen/tecsgen/tecslib/core/componentobj.rb	Tue May 31 01:59:04 2016
+++ tecsgen/tecsgen/tecslib/core/componentobj.rb	Tue May 31 01:59:04 2016
@@ -5752,6 +5752,11 @@
     @cell = c
     # @definition = nil          # @definition が有効： チェック済み（とは、しない）
 
+    # 受け口配列
+    if @rhs_subscript then
+      @cell.set_entry_port_max_subscript( @port, @rhs_subscript )
+    end
+
     if @array_member2 then
 
       # debug
